### PR TITLE
Logout and Attack Dashboard prep

### PR DESCRIPTION
**Pull request by Aviv-Galmidi**
Add logout button to the side menu
Fix logout for user idle modal
Change Default Dashboard filters to be inside that dashboard
Add prep work for Attack Dashboard
Fix issue with duplicated entries in Global Filters not being caught correctly

### CODE REVIEW

File: src/reblaze-dashboards/AttackDashboard.vue [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+export default defineComponent({
+  name: 'RbzDashboardAttack',
+  components: {
+    RbzDatePicker,
+  },
+  data() {
+    return {
+      isSearchLoading: false,
+      data: [] as any[], // TODO
```
Comment (AvihayMReblaze) [inline src/reblaze-dashboards/AttackDashboard.vue:110]: Remove // TODO

File: src/reblaze-dashboards/AttackDashboard.vue [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+                  <i class="fas fa-times"></i>
+                </span>
+                <span>
+                  Clear
+                </span>
+              </button>
+            </div>
+          </div>
+        </div>
+        <!--TODO: Remove empty div, fix element position-->
```
Comment (AvihayMReblaze) [inline src/reblaze-dashboards/AttackDashboard.vue:49]: Remove TODO

File: src/reblaze-dashboards/AttackDashboard.vue [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+            </span>
+            <span>
+              Filter
+            </span>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div v-show="!isSearchLoading && data?.length > 0">
+      <!--TODO: Graphs and tables-->
```
Comment (AvihayMReblaze) [inline src/reblaze-dashboards/AttackDashboard.vue:75]: Remove TODO

**Review by AvihayMReblaze — CHANGES_REQUESTED** [review]

File: src/reblaze-dashboards/AttackDashboard.vue (hunk 1 above)
Comment (Aviv-Galmidi) [reply src/reblaze-dashboards/AttackDashboard.vue:110]: This is an actual TODO that needs to be done in the future, the Attack Dashboard isn't done but it's prep work for it

File: src/reblaze-dashboards/AttackDashboard.vue (hunk 2 above)
Comment (Aviv-Galmidi) [reply src/reblaze-dashboards/AttackDashboard.vue:49]: This is an actual TODO that needs to be done in the future, the Attack Dashboard isn't done but it's prep work for it

File: src/reblaze-dashboards/AttackDashboard.vue (hunk 3 above)
Comment (Aviv-Galmidi) [reply src/reblaze-dashboards/AttackDashboard.vue:75]: This is an actual TODO that needs to be done in the future, the Attack Dashboard isn't done but it's prep work for it

**Review by AvihayMReblaze — APPROVED** [review]

**Pull request merged**